nbproject/__init__.py

```diff
@@ -25,16 +25,6 @@
 """
 __version__ = "0.10.1"
 
-# init jupyter lab frontend immediately on import
-# nothing happens if this is not jupyter lab
-from .dev._jupyter_lab_commands import _init_frontend
-
-# trying to init ipylab JupyterFrontEnd can lead to errors on jupyter notebook
-try:
-    _init_frontend()
```

Comment:
I don't agree with this, this is here for a good resaon and definitely should be called if ipylab is installed.

Reply:
I don't very much mind if this is being reverted if it's not preventing to eliminate ipylab:

- https://github.com/laminlabs/nbproject/issues/281

I removed it because I thought it may cause latency due to the error.

Alternatively, we port everything we need from nbproject into lamindb and no longer rely on nbproject.

But to be honest, I don't think anybody uses these interactive features, so, I don't think it's worth investing anytime that's not meant to streamline lamindb (make lamindb higher quality, faster, more lightweight, etc.).

Reply:
The problem is that ipylab is used not only for interactive features, but also for detecting the notebook path if everything else fails, this is relevant for some cloud services and we should add a warning in such cases.

Reply:
Yes, I saw this! If it's hitting it's now going to error with `ipylab` isn't installed. And if this is actually occurring somewhere, we'll then learn about it.

The logic is still there! I just made ipylab optional.

Reply:
But it should not error in this case, it should till try the environment variable. Ok, i will make a PR.

Reply:
Yes, please, just make a PR that restores what you think should be there.

I also think it's perfectly fine if nbproject has an `ipylab` extra, but I'd prefer to not install it through `lamindb[jupyter]`.